modules/acquia_cms_common/src/RedirectHandler.php

```diff
@@ -119,12 +131,14 @@ protected function handleRedirect(FormStateInterface $form_state): void {
       // is fixed.
       $this->request->query->remove('destination');
 
-      if ($this->isContributor($user)) {
-        // @todo Don't redirect if Moderation Dashboard is not enabled.
+      if ($this->isContributor($user) &&
+      $this->moduleHandler->moduleExists('moderation_dashboard') &&
+      $this->configFactory->getEditable('moderation_dashboard.settings')->get('redirect_on_login')) {
```

Comment:
I think we should use `$this->configFactory->get('moderation_dashboard.settings')->get('redirect_on_login')` instead to respect the overrides.

Reply:
Done